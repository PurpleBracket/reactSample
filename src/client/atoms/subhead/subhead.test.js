/* global describe test expect */
import React from 'react'
import { shallow } from 'enzyme'
import Subhead from './subhead'

describe('Subhead', () => {
  test('should match snapshot', () => {
    const wrapper = shallow(<Subhead size='1'>Test Subhead</Subhead>)
    expect(wrapper).toMatchSnapshot()
  })
})
