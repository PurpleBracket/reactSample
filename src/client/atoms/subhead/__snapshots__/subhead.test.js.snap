// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Subhead should match snapshot 1`] = `
<h1
  className="subtitle subhead is-1 "
  style={Object {}}
>
  Test Subhead
</h1>
`;
